docs/mysql.md

```diff
@@ -1,3 +1,75 @@
+# `mysql` Metrics Receiver
+
+The mysql receiver can retrieve stats from your mysql instance by connecting as a monitoring user.
+
+
+## Prerequisites
+
+It is recommended that you create a dedicated monitoring user. The monitoring user must be granted the following permissions:
```

Comment:
Can you add before this something like:

```md
The `mysql` receiver defaults to connecting to a local MySQL server using a Unix socket and Unix authentication as the `root` user.
```

to make the default behavior clear?

Reply:
added

Reply:
Thanks. Why do we recommend a dedicated monitoring user? I think I would phrase it as "If you want to create a dedicated monitoring user, the monitoring user must be granted ..."

Reply:
The reasoning behind it is so they wouldn't be giving root access to everything with the default config. But I'll update the wording if you think its better?

Reply:
@quentinmit WDYT?

Reply:
Ops agent is currently running as root on the machine, so it already has de facto root access to the local MySQL server. So I don't think there's much risk mitigation from telling users to create another user (and that would mean using passwords, which are less secure than local authentication).

Reply:
The sentence in question has been removed.